src/libsystemd/sd-journal/journal-file.c

```diff
@@ -2757,7 +2757,7 @@ static int generic_array_get(
                 Object **ret_object,    /* The found object. */
                 uint64_t *ret_offset) { /* The offset of the found object. */
 
-        uint64_t a, t = 0, k;
+        uint64_t a, t = 0, k = 0;
```

Comment:
Could you also add a short comment for the initialization of `k` ?

Reply:
Hopefully it's the last one (at least testing this locally on Ubuntu Jammy finally made gcc happy).